src/util/config.tsx

```diff
@@ -39,7 +39,7 @@ export const configDescriptionToHtml = (
   // documentation links
   if (objectsInvTxt) {
     // tags like {ref}`instance-options-qemu` can be in the input string
-    const linkTags = input.match(/{(config|ref|config:option)}`[a-z-:.]+`/g);
+    const linkTags = input.match(/{(config|ref|config:option)}`[a-z-:._]+`/g);
```

Comment:
I wonder if there are other edge cases coming from sphinx

Reply:
This is what is being used as tag name in the LXD code. I think we can extend it if new ones arise.